### DISCUSSION TITLE
Who can see organization contact email?

### DISCUSSION THREAD

**Discussion by Chilichurl** [community/community#30668] [New to GitHub] [answered]
Hello everyone!
When creating a new organization, there is a required field "Contact email". 
Is this a publicly available email that anyone can use to contact the organization? 
Or is it private and only GitHub stuff can use it, for security messages and important info? 
Or is it visible only to the members of organization?
I'm not sure which existing email would be best to use or if I should create a new dedicated one just for it.

**Comment by melvincwng** [comment, marked as answer]
Hey @Chilichurl, if I'm not mistaken, the organization email should be a public field.

Go to your organization > Settings > General > You should see a 'Contact Email' field that shows that it is public

Example (taken from my very own organization):
<img src="https://user-images.githubusercontent.com/77479885/186435859-a9fac0aa-ac61-48c7-92e8-afb855d3e682.png" alt="example_image" width="600" height="300"/>

Hope this answers your question 😄

**Reply by Chilichurl** [reply]
Thank you very much!

**Reply by melvincwng** [reply]
@Chilichurl, pleasure in helping ^^

**Comment by RobotoLev** [comment]
The answer is kind of incorrect nowadays. "Contact email" field that must be set up during the org creation sets a "Billing email" in the org settings, which is a private email.

**Comment by alonso-fernandezr** [comment]
Hey! That error usually happens because of network issues or firewall blocking. 
First, make sure your internet connection is stable. 
Then try pinging `github.com` from your terminal to check connectivity. If you're using SSH, confirm your SSH keys are set up correctly. 
Also, sometimes a VPN or proxy can cause issues, so try disabling them if you have one. 
Hope this helps!
